src/pages/AppLogin.vue

```diff
@@ -74,22 +74,29 @@
 
 <script>
 import { onMounted } from "vue";
+import { useRouter } from "vue-router";
 
 import useArcanaWallet from "../use/arcanaWallet";
 import useToast from "../use/toast";
 
 export default {
   setup() {
-    const { toastError } = useToast();
-    const { requestSocialLogin } = useArcanaWallet();
+    const router = useRouter();
+    const { toastSuccess, toastError } = useToast();
+    const { requestSocialLogin, setHook } = useArcanaWallet();
 
     onMounted(async () => {
       document.title = "Login | Arcana Demo";
+      setHook("accountsChanged", (params) =>
+        console.log({ type: "accountsChanged", params })
+      );
     });
 
     async function onSignInClick() {
       try {
         await requestSocialLogin("google");
+        await router.push("/my-files");
+        toastSuccess("Login Success");
```

Comment:
Since we come back to the same page after a successful login, this router push is required to move to the my files page.